api/v1beta1/openstackvmset_types.go

```diff
@@ -31,12 +31,10 @@ type OpenStackVMSetSpec struct {
 	Memory uint32 `json:"memory"`
 	// root Disc size in GB
 	DiskSize uint32 `json:"diskSize"`
-	// Name of the VM base image used to setup the VMs
-	BaseImageURL string `json:"baseImageURL,omitempty"`
 	// StorageClass to be used for the disks
 	StorageClass string `json:"storageClass,omitempty"`
-	// BaseImageVolumeName Optional. If supplied will be used as the base volume for the VM instead of BaseImageURL.
-	BaseImageVolumeName string `json:"baseImageVolumeName,omitempty"`
+	// BaseImageVolumeName Optional. used as the base volume for the VM
```

Comment:
Should probably take "optional" out of the description

Reply:
yes, lets remove optional and ready to be merged

Reply:
got it